### ISSUE TITLE
remove event handler method

### ISSUE THREAD

**Issue by ramjak** [ClusterWS/ClusterWS#127]
### Submitting

- [ ] Bug
- [ ] Question
- [x] Suggestion
- [ ] Other

### Details 
It would be nice if this library has remove event handler method. Maybe the syntax would be
`socket.off(eventName)` since there is `socket.on(eventName, handler)` to connect event. And also it would be compatible with rxjs.fromEvent method.

**Comment by ramjak** [comment]
I've also make a commit for ClusterWS-Client-js [here](https://github.com/ramjak/ClusterWS-Client-JS/commit/d6f14ab4790b5ae61473ffd809b8a320ac53ab79), but the builder couldn't run as I expected in my machine.

**Comment by goriunov** [comment]
Yes ClusterWS-Client-js  has very old build system which got broken with latest packages (i am going to change it when i start working on new version of the client) it does have `removeEvent` but yes i am agree it would be good to rename that to `off` as from node js 10 seems to have it as one of the alliance. 

Would you mind doing a pull request for that ? (file is in `src/utils/emitter`).

About your pull `ClusterWS-Client-JS` i will write there :)

**Comment by ramjak** [comment]
Of course, it's here: https://github.com/ClusterWS/ClusterWS-Client-JS/pull/52. Thanks for the quick follow-up
